DemocracyClub/WhoCanIVoteFor
# Feature/add more election results data

Closes https://github.com/DemocracyClub/WhoCanIVoteFor/issues/935 and https://github.com/DemocracyClub/WhoCanIVoteFor/issues/1663
Dependent on https://github.com/DemocracyClub/yournextrepresentative/pull/2174

This work adds election results for past elections, when results are available.

- [x] Add turnout, ballot papers issues, electorate and spoilt ballots to `Election` model
- [x] Display ^^ results data in a new table on the ballot template as shown below, only when they exist
- [x] Add Welsh translations
- [x] Import this data as part of the `import_ballots` process
- [x] Add election results to YNR ballot serializer
- [x] Switches candidate lists to a single column

To test:
- run migrations
- run the ballot importer for recent elections with `python manage.py import_ballots --current`
- visit a [recently past election](http://localhost:8000/elections/local.st-albans.marshalswick-east-jersey-farm.by.2023-08-17/marshalswick-east-jersey-farm/) with results to view a new table similar to the one shown below

![Screenshot 2023-09-19 at 4 41 03 PM](https://github.com/DemocracyClub/WhoCanIVoteFor/assets/7017118/ba9a5ce6-6339-4a76-bce9-70cb38782dff)

![Screenshot 2023-09-19 at 4 41 19 PM](https://github.com/DemocracyClub/WhoCanIVoteFor/assets/7017118/adc1523c-4b70-4725-b831-5a02741a6d08)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205303544790826
